package.json

```diff
@@ -4,6 +4,7 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
```

Comment:
Has coveralls been able to display coverage data?

Reply:
Yes.
https://coveralls.io/github/andela/rogue-backend?branch=ch-integrate-coveralls-167727620